app/services/cms/models/StrapiBox.ts

```diff
@@ -1,36 +1,25 @@
-import pick from "lodash/pick";
 import { z } from "zod";
-import type { BoxProps } from "~/components/Box";
-import { omitNull } from "~/util/omitNull";
 import { HasOptionalStrapiIdSchema } from "./HasStrapiId";
 import { OptionalStrapiLinkIdentifierSchema } from "./HasStrapiLinkIdentifier";
 import { StrapiBackgroundSchema } from "./StrapiBackground";
 import { StrapiButtonSchema } from "./StrapiButton";
 import { StrapiContainerSchema } from "./StrapiContainer";
 import { StrapiHeadingSchema } from "./StrapiHeading";
-import { StrapiParagraphSchema, getRichTextProps } from "./StrapiParagraph";
+import { getRichTextProps, StrapiParagraphSchema } from "./StrapiParagraph";
 
-const StrapiBoxSchema = z
+export const StrapiBoxSchema = z
   .object({
-    label: StrapiHeadingSchema.nullable(),
-    heading: StrapiHeadingSchema.nullable(),
+    __component: z.literal("page.box"),
+    label: StrapiHeadingSchema.optional(),
+    heading: StrapiHeadingSchema.optional(),
```

Comment:
i think these should be defined as 
```suggestion
    heading: StrapiHeadingSchema.nullable().transform(omitNull).optional(),
```

Reply:
i think we should use optional instead of converting to nullable and then transforming the null to undefined.
optional returns value | undefined and it looks more cleaner, from my POV

Reply:
unfortunately we have to define them first with `nullable`, as that's what exists in Strapi; Strapi has no conception of `optional`; that gets added on after schema validation + transformation